app/models/ems_refresh/refreshers/ems_refresher_mixin.rb

```diff
@@ -90,9 +91,20 @@ def refresh_targets_for_ems(ems, targets)
 
           Benchmark.realtime_block(:save_inventory) { save_inventory(ems, target, hashes) }
           _log.info "#{log_header} Refreshing target #{target.class} [#{target.name}] id [#{target.id}]...Complete"
+
+          if hashes.kind_of?(Array)
+            _log.info "#{log_header} ManagerRefresh Post Processing #{target.class} [#{target.name}] id [#{target.id}]..."
+            # We have array of InventoryCollection, we want to use that data for post refresh
+            Benchmark.realtime_block(:manager_refresh_post_processing) { manager_refresh_post_processing(ems, target, hashes) }
+            _log.info "#{log_header} ManagerRefresh Post Processing #{target.class} [#{target.name}] id [#{target.id}]...Complete"
+          end
         end
       end
 
+      def manager_refresh_post_processing(_ems, _target, _inventory_collections)
```

Comment:
From what I see we can't, the current post refresh is called to late, so at a point where all InventoryCollections are thrown away. And we cannot simply store them, since it goes over many targets, so there is possibility of having more Persiters there